src/ray/protobuf/gcs.proto

```diff
@@ -27,7 +27,8 @@ enum TablePrefix {
   ACTOR_CHECKPOINT_ID = 16;
   NODE_RESOURCE = 17;
   DIRECT_ACTOR = 18;
-  TABLE_PREFIX_MAX = 19;
+  WORKER_FAILURE = 19;
```

Comment:
Hmm maybe we should publish node and worker failure events on the same channel? They have a similar schema: {required node_id, required ip, optional worker_id, optional worker_port}.

Reply:
True, that's a good idea. Originally I was thinking about doing it from the client table but it does seem better to consolidate. I'll add this.

Reply:
Hmm actually it seems a bit annoying because the ClientID and WorkerIDs are different types...I'll leave a TODO.